src/commands/azure/deleteImage.ts

```diff
@@ -6,58 +6,19 @@
 import { Registry } from "azure-arm-containerregistry/lib/models";
 import * as vscode from "vscode";
 import { DialogResponses, IActionContext } from "vscode-azureextensionui";
-import { AzureImageTagNode } from '../../explorer/models/azureRegistryNodes';
+import { AzureImageTagNode } from '../../../explorer/models/azureRegistryNodes';
 import { ext } from "../../extensionVariables";
 import * as acrTools from '../../utils/Azure/acrTools';
-import { AzureImage } from "../../utils/Azure/models/image";
-import { Repository } from "../../utils/Azure/models/repository";
-import * as quickPicks from '../utils/quick-pick-azure';
-
-/** Function to untag an Azure hosted image
- * @param node : if called through right click on AzureImageNode, the node object will be passed in. See azureRegistryNodes.ts for more info
- */
-export async function untagAzureImage(_context: IActionContext, node?: AzureImageTagNode): Promise<void> {
-    let registry: Registry;
-    let repo: Repository;
-    let image: AzureImage;
-
-    if (!node) {
-        registry = await quickPicks.quickPickACRRegistry();
-        repo = await quickPicks.quickPickACRRepository(registry, `Select the repository of the image you want to untag`);
-        image = await quickPicks.quickPickACRImage(repo, `Select the image you want to untag`);
-
-    } else {
-        registry = node.registry;
-        let wholeName: string[] = node.label.split(':');
-        repo = await Repository.Create(registry, wholeName[0]);
-        image = new AzureImage(repo, wholeName[1]);
-    }
-
-    const untag: vscode.MessageItem = { title: "Untag" };
-    const shouldDelete = await ext.ui.showWarningMessage(
-        `Are you sure you want to untag '${image.toString()}'? This does not delete the manifest referenced by the tag.`,
-        { modal: true },
-        untag,
-        DialogResponses.cancel);
-
-    if (shouldDelete === untag) {
-        await acrTools.untagImage(image);
-        vscode.window.showInformationMessage(`Successfully untagged '${image.toString()}'`);
-
-        if (node) {
-            ext.dockerExplorerProvider.refreshNode(node.parent);
-        } else {
-            ext.dockerExplorerProvider.refreshRegistries();
-        }
-    }
-}
+import { AzureImage } from "../../utils/Azure/models/AzureImage";
+import { AzureRepository } from "../../utils/Azure/models/AzureRepository";
+import * as quickPicks from '../../utils/quick-pick-azure';
 
 /** Function to delete an Azure hosted image
  * @param node : if called through right click on AzureImageNode, the node object will be passed in. See azureRegistryNodes.ts for more info
  */
-export async function deleteAzureImage(_context: IActionContext, node?: AzureImageTagNode): Promise<void> {
+export async function deleteImage(_context: IActionContext, node?: AzureImageTagNode): Promise<void> {
```

Comment:
If it's deleting an Azure image, why rename the function?

Reply:
Since I was renaming all the files anyways, I turned on the "export-name" tslint rule. The file for `deleteAzureImage` was `delete-image`. I don't have a strong preference on the name, but if you look at the ACR commands, none of them had "Azure" in their file name so that's what I went with. Here's all the command files currently on master: https://github.com/microsoft/vscode-docker/tree/master/commands/azureCommands

Reply:
Personally I'd prefer you to keep the function names with Azure and change the filenames to match. Makes it easier when reading in context.

Reply:
Fixed